sdks/python/setup.py

```diff
@@ -368,7 +372,6 @@ def get_portability_package_data():
           'grpcio>=1.33.1,!=1.48.0,<2',
           'hdfs>=2.1.0,<3.0.0',
           'httplib2>=0.8,<0.23.0',
-          'js2py>=0.74,<1',
```

Comment:
we can simplify it via an environmental marker: https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#platform-specific-dependencies

Reply:
Also let's reference the issue in JS2Py's tracker

Reply:
Thanks for the heads up - refactored to use environmental marker.